phase3/database.py

```diff
@@ -75,3 +77,21 @@ def retrieve_one(self, image_id):
         connection.close()
 
         return query_result
+    
+    def insert_binary(self, key, value, collection_type=None):
+        connection = self.open_connection()
```

Comment:
For `insert_binary` and `retrieve_binary` you can remove `collection_type` as I dont think these methods will be used anywhere other than  ppr